mods/default/models/torch_floor.obj

```diff
@@ -0,0 +1,48 @@
+# Blender v2.73 (sub 0) OBJ File: 'mt_torch_full_vertices3.blend'
+# www.blender.org
+mtllib mt_torch_full_vertices3.mtl
+o Cube_Cube.001
+v 0.061653 -0.496733 0.061674
+v 0.061653 -0.496733 -0.062460
+v -0.059258 -0.496733 0.061674
+v -0.059258 -0.496733 -0.062460
+v -0.059994 -0.497234 0.497315
+v -0.059994 -0.497234 -0.498102
+v -0.059994 0.507706 -0.498102
+v -0.059994 0.507706 0.497315
+v -0.494900 0.507706 0.061709
+v -0.494900 -0.497234 0.061709
+v 0.497295 0.507706 0.061709
+v 0.497295 -0.497234 0.061709
+v 0.062686 0.507706 -0.498102
+v 0.062686 -0.497234 -0.498102
+v 0.062686 0.507706 0.497315
+v 0.062686 -0.497234 0.497315
+v -0.494900 0.507706 -0.063149
+v -0.494900 -0.497234 -0.063149
+v 0.497295 0.507706 -0.063149
+v 0.497295 -0.497234 -0.063149
+v -0.058217 0.134520 0.060216
+v -0.058217 0.135872 -0.061813
+v 0.061944 0.135872 -0.061813
+v 0.061944 0.134520 0.060216
```

Comment:
Don't get me wrong. The vertice count isn't actually why i'm asking. It's just that i'd expect some weird graphical intersecting going on with some texturesets.